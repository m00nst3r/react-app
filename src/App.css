.App {
  /*text-align: center;*/
  padding-top: 20px;
}

.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 80px;
}

.App-header {
  background-color: #222;
  height: 150px;
  padding: 20px;
  color: white;
}

.App-intro {
  font-size: large;
}

@keyframes App-logo-spin {
  from { transform: rotate(0deg); }
  to { transform: rotate(360deg); }
}

.drop-down {
  color: #c7caca;
}

.App-Block {
  position: relative;
  height: 150px;
  overflow: hidden;
  border: 1px solid #c7caca;
  display: block;
  z-index: auto;
}

.App-Block .Select {
  position: fixed;
  display: inline-block;
  width: 300px;
  z-index: 1;
}

.second {
  background-color: #fff;
  height: 300px;
}
.box-title {
  padding: 10px 0 0 10px;
  display: block;
  background: rgb(0, 0, 94);
  font-weight: bold;
  color: #ffffff;
  border-bottom: 4px solid rgb(64, 191, 163);
  text-align: left;
}
.box-body {
  background-color: #fff;
}

.action-keys {
  display: inline-block;
  font-weight: normal;
  /*text-align: right;*/
}
.action-keys i{
  cursor: pointer;
}

.navigation {
  position: absolute;
  z-index: 1;
  top: 0;
  background-color: rgb(0, 0, 94);
  border-bottom: 4px solid rgb(64, 191, 163);
  width: 100%;
  padding: 6px 0;
}
.actions-bar {
  display: inline-block;
}
.actions-bar i {
  color: #ffffff;
  margin: 4px 0 0 2px;
  cursor: pointer;
}
.single-value {
  padding-top: 30px;
}
.single-value-title {
  display: flex;
  flex-direction: row;
  align-items: center;
  padding: 15px;
  font-size: 14px;
  color: white;
  border-bottom: 1px dashed #344B6D;
}
.single-value-title small {
  color: #6086d6;
}
.single-value-chart {
  width: 400px;
  height: 150px;
  background: #27273f;
  border-radius: 8px;
  color: #00f1a1;
  display: flex;
  flex-direction: column;
  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.7);
}
.single-value-body {
  font-size: 40px;
  padding: 15px;
  text-align: center;
}
.list-app {
  padding-top: 25px;
}
.filters .Select{
  width: 200px;
  display: inline-block;
  margin-right: 10px;
}

.filter-item {
  display: inline-block;
}
.filter-item h1 {
  font-size: 12px;
  font-weight:bold;
}
.table-limit th {
  padding: 14px;
}

.table-limit td {
  padding-top: 10px;
}