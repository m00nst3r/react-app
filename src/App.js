import React, {Component} from 'react';
import PropTypes from 'prop-types'
import injectTapEventPlugin from 'react-tap-event-plugin';
import Navigation from "./components/Navigation";


class App extends Component {
  componentWillMount() {
    injectTapEventPlugin()
  }



  render() {
    return (
      <div className="App">
        <Navigation/>
        {this.props.children}
      </div>
    );
  }
}

App.propTypes = {
  children: PropTypes.node
};

export default App;
