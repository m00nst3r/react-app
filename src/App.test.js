import React from 'react';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import App from './App';

it('renders without crashing', () => {
  const component = renderer.create(<App/>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
  // const div = document.createElement('div');
  // ReactDOM.render(<App />, div);
});
