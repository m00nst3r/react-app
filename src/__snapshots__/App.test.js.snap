// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="App"
>
  <div
    className="navigation"
  >
    <span>
      <a
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
       | 
    </span>
    <span>
      <a
        onClick={[Function]}
        style={Object {}}
      >
        Test application
      </a>
       | 
    </span>
    <span>
      <a
        onClick={[Function]}
        style={Object {}}
      >
        List Application
      </a>
       | 
    </span>
    <span>
      <a
        onClick={[Function]}
        style={Object {}}
      >
        Single Value
      </a>
       | 
    </span>
    <span>
      <a
        onClick={[Function]}
        style={Object {}}
      >
        Resize
      </a>
       | 
    </span>
    <span>
      <a
        onClick={[Function]}
        style={Object {}}
      >
        Block
      </a>
       | 
    </span>
  </div>
</div>
`;
